File: 1160 - Crescimento Populacional/crescimento_populacional.py
t = int(input())
for _ in range(t):
    partes = input().split()
    pa, pb = int(partes[0]), int(partes[1])
    g1, g2 = float(partes[2]), float(partes[3])
    anos = 0
    while pa <= pb and anos < 101:
        pa += int(pa * g1 / 100)
        pb += int(pb * g2 / 100)
        anos += 1
    if anos > 100:
        print("Mais de 1 seculo.")
    else:
        print(f"{anos} anos.")

File: 1160 - Crescimento Populacional/crescimento_populacional.c
#include <stdio.h>

int main(){
    int t;
    scanf("%d", &t);
    while(t--){
        long long pa, pb;
        double g1, g2;
        scanf("%lld %lld %lf %lf", &pa, &pb, &g1, &g2);
        int anos = 0;
        while(pa <= pb && anos < 101){
            pa += (long long)(pa * g1 / 100);
            pb += (long long)(pb * g2 / 100);
            anos++;
        }
        if(anos > 100) printf("Mais de 1 seculo.\n");
        else printf("%d anos.\n", anos);
    }
    return 0;
}
